modules/alacritty/hm.nix

```
@@ -1,31 +1,60 @@
-{pkgs, config, lib, ... }:
-
-with config.stylix.fonts;
+# Documentation is available at https://alacritty.org/config-alacritty.html.
+{ config, lib, ... }:
 
 let
-  useYaml = (builtins.compareVersions config.programs.alacritty.package.version "0.13.0") < 0;
-  templateRepo = config.lib.stylix.templates.
-    "base16-alacritty${if useYaml then "-yaml" else ""}";
-
-  themeFile = config.lib.stylix.colors {
-    inherit templateRepo;
-  };
+  colors = config.lib.stylix.colors;
+  mnemonicsWithHashtag = builtins.mapAttrs (_: color: "#${color}") colors;
 in
 {
-  options.stylix.targets.alacritty.enable =
-    config.lib.stylix.mkEnableTarget "Alacritty" true;
+  options.stylix.targets.alacritty.enable = config.lib.stylix.mkEnableTarget "Alacritty" true;
 
   config = lib.mkIf config.stylix.targets.alacritty.enable {
     programs.alacritty.settings = {
-      font = {
+      font = with config.stylix.fonts; {
         normal = {
           family = monospace.name;
           style = "Regular";
         };
         size = sizes.terminal;
       };
       window.opacity = with config.stylix.opacity; terminal;
-      import = [ themeFile ];
+      colors = with colors.withHashtag; {
+        primary = {
+          foreground = base05;
+          background = base00;
+          bright_foreground = base07;
+        };
+        selection = {
+          text = base05;
+          background = base02;
+        };
+        cursor = {
+          text = base00;
+          cursor = base05;
+        };
+        normal = {
+          black = base00;
+          white = base05;
+          inherit (mnemonicsWithHashtag)
+            red
+            green
+            yellow
+            blue
+            magenta
+            cyan
+            ;
```

This would be fine to write in one line.

```suggestion
          inherit (mnemonicsWithHashtag) red green yellow blue magenta cyan;
```

---

Done. All of the code for this module has been formatted with rfc-style nix-fmt. Long inherits is the one area it's sub-par atm.